metricflow/sql/render/postgres.py

```diff
@@ -44,6 +47,32 @@ def visit_generate_uuid_expr(self, node: SqlGenerateUuidExpression) -> SqlExpres
             execution_parameters=SqlBindParameters(),
         )
 
+    def visit_percentile_expr(self, node: SqlPercentileExpression) -> SqlExpressionRenderResult:
+        """Render a percentile expression for Postgres."""
+        arg_rendered = self.render_sql_expr(node.order_by_arg)
+        params = arg_rendered.execution_parameters
+        percentile = node.percentile_args.percentile
+
+        if node.percentile_args.function_type is SqlPercentileFunctionType.CONTINUOUS:
+            function_str = "PERCENTILE_CONT"
+        elif node.percentile_args.function_type is SqlPercentileFunctionType.DISCRETE:
+            function_str = "PERCENTILE_DISC"
+        elif node.percentile_args.function_type is SqlPercentileFunctionType.APPROXIMATE_CONTINUOUS:
+            raise RuntimeError(
+                "Approximate continuous percentile aggregate not supported for Postgres. Set use_approximate_percentile to false in all measures."
+            )
+        elif node.percentile_args.function_type is SqlPercentileFunctionType.APPROXIMATE_DISCRETE:
+            raise RuntimeError(
+                "Approximate discrete percentile aggregate not supported for Postgres. Set use_approximate_percentile to false in all measures."
+            )
```

Comment:
Same thing on this and all other error messages re: line length and increased clarity.